src/extension.ts

```diff
@@ -72,9 +73,24 @@ export function activate(context: vscode.ExtensionContext) {
 
     context.subscriptions.push(disposable);
 
-    disposable = vscode.commands.registerCommand('extension.ccUpdateDir', () => {
+    disposable = vscode.commands.registerCommand('extension.ccUpdateView', () => {
         if ( vscode.workspace.rootPath )
-            updatePath(vscode.workspace.rootPath);
+            updateObject(vscode.workspace.rootPath);
+    });
+
+    disposable = vscode.commands.registerCommand('extension.ccUpdateDir', () => {
+        if ( vscode.window &&
+             vscode.window.activeTextEditor &&
+             vscode.window.activeTextEditor.document ) {
+            updateObject(dirname(vscode.window.activeTextEditor.document.fileName));
```

Comment:
I can have a look how to integrate such a command in the tree view context menu.